### ISSUE TITLE
Support passing arguments to ctr tasks

### ISSUE THREAD

**Issue by jvalkeal** [spring-cloud/spring-cloud-dataflow#4265]
Currently arguments with composed task are just passed to ctr itself and its `composed-task-arguments` option passes same arguments to all tasks. 

If having task `t1: timestamp && t2: timestamp` with name `ctr1` we could use similar format than in other parts:
```
app.*=--a=a
app.*=--d=d
app.t1=--b=b
app.t1=--e-e
app.t2=--c=c
app.t2=--f=f
```
`app.<task label>=` would be optional and if not present it'd be just a normal argument going into ctr app itself or to a task in case of simple task(non ctr).

Basically this new format means that if an argument start with `app.ddd=` it would be handled as an special option meant to ctr app itself. There's no need to think breaking changes as any these types of an arguments would have been discarded by ctr app itself.

**Comment by cppwfs** [comment]
I agree with the idea.   
We have discussed this before. But I don't see where we created a Issue for it in either Data Flow or the old CTR project.  
To also clarify that this will require a change to both data flow (to validate the command line args before passing to CTR) and CTR to pass the args to the appropriate apps. 
One question: app.ctr1.*=--a=a means that `--a=a` will be applied to all apps in the flow, correct?
